File: BackEnd/StraviaTECAPI/StraviaTECAPI/Models/UsuarioRol.cs
using System;
using System.Collections.Generic;

namespace StraviaTECAPI.Models
{
    public partial class UsuarioRol
    {
        public int IdUsuario { get; set; }
        public int IdRol { get; set; }

        public Rol IdRolNavigation { get; set; }
        public Usuario IdUsuarioNavigation { get; set; }
    }
}

File: FrontEnd/Angular/src/app/Models/UsuarioRol.js
export interface UsuarioRol{
    
    IdUsuario: Number,
    IdRol: Number,
    IdRolNavigation: [
        Id: Number,
        Nombre: String
    ],
    IdUsuarioNavigation: [
        cedula: Number,
        nombre: String,
        apellido: String,
        nombreUsuario: String,
        contraseña: String,
        nacionalidad: String,
        fechaNacimiento: String,
        Foto: ByteLengthChunk //no se si esto esta bien
    ]
}
